app/widgets/common/deployments/DeploymentActionsMenu.utils.ts

```diff
@@ -4,7 +4,7 @@ import type { MenuItem } from './DeploymentActionsMenu.consts';
 import { permissions, actions } from './DeploymentActionsMenu.consts';
 
 function hasEnvironmentLabel(deploymentLabels: Label[]) {
-    return deploymentLabels.some(deploymentLabel => {
+    return deploymentLabels?.some(deploymentLabel => {
         return deploymentLabel.key === 'csys-obj-type' && deploymentLabel.value === 'environment';
     });
 }
```

Comment:
WDYT about the following modification?

```suggestion
function hasEnvironmentLabel(deploymentLabels: Label[] = []) {
    return deploymentLabels.some(deploymentLabel => {
        return deploymentLabel.key === 'csys-obj-type' && deploymentLabel.value === 'environment';
    });
}
```

Reply:
[b6662f3](https://github.com/cloudify-cosmo/cloudify-stage/pull/2496/commits/b6662f38f932f90939227d8e6e09984dbe6cf7d5)